packages/helix/project.bri

```diff
@@ -0,0 +1,48 @@
+import * as std from "std";
+import git from "git";
+import rust, { cargoBuild } from "rust";
+
+export const project = {
+  name: "helix",
+  version: "25.01.1",
+  repository: "https://github.com/helix-editor/helix.git",
+};
+
+const source = Brioche.gitCheckout({
+  repository: project.repository,
+  ref: project.version,
+});
+
+export default function helix(): std.Recipe<std.Directory> {
+  return cargoBuild({
+    env: {
+      HELIX_DISABLE_AUTO_GRAMMAR_BUILD: "1",
+    },
+    path: "helix-term",
+    runnable: "hx",
```

Comment:
To see the actual output of the package built, you can run `brioche build -o /tmp/output -p packages/helix`, and in the folder output, you can see:

```bash
> ls /tmp/output
bin  brioche-resources.d  brioche-run
> ls /tmp/output/bin
hx
```

```suggestion
    runnable: "bin/hx",
```